spacelift/resource_stack.go

```diff
@@ -61,6 +61,36 @@ func resourceStack() *schema.Resource {
 				Description: "GitHub branch to apply changes to",
 				Required:    true,
 			},
+			"cloudformation": {
```

Comment:
I think `ConflictsWith` would not hurt here?

Reply:
Oh yeah, thanks, forgot about it.